cpp/src/slice2cs/Gen.cpp

```diff
@@ -679,7 +667,7 @@ Slice::CsVisitor::getDispatchParams(
         retS = typeToString(op->returnType(), ns, op->returnIsOptional());
     }
 
-    string currentParamName = getEscapedParamName(op, "current");
+    string currentParamName = getEscapedParamName(op->parameters(), "current");
```

Comment:
`->parameters()` means all parameters. I kept the old logic; it's likely we could do better.